### DISCUSSION TITLE
Welcome to community Discussions!

### DISCUSSION THREAD

**Discussion by norrec99** [near-in-minutes/community#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ozanisgor** [comment]
test

Cagatay Soyer ***@***.***>, 8 Haz 2021 Sal, 15:43 tarihinde
şunu yazdı:

**Reply by hiba-machfej** [reply]
Thank you for creating this @norrec99
